deepmd/common.py

```diff
@@ -67,9 +67,13 @@ def gelu(x: tf.Tensor) -> tf.Tensor:
     """
     def gelu_wrapper(x):
         try:
-            return tensorflow.nn.gelu(x, approximate=True)
+            tensorflow.nn.gelu
```

Comment:
What is the influence of using approximate=False instead of the approximate=True ? 
How does it get the env `DP_JIT_LEVEL`?

Reply:
This line is to check if TF supports gelu... The actual line is 74-76.

> How does it get the env DP_JIT_LEVEL?
line 76 does such things.

Reply:
I see Thanks!